src/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  box-sizing: border-box;
+  --background-color: #fff;
+  --second-background-color: #00acdc;
+  --text-color: #060b35;
+  --second-text-color: #616070;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__imac {
+  margin-top: 32px;
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__text {
+  margin: 40px 0 16px;
+}
+
+.card__text_monoblock {
+  margin: 0;
+  margin-left: calc(50% - 166px / 2);
+  display: flex;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--text-color);
+  line-height: 18px;
+}
+
+.card__text_code {
+  margin-top: 4px;
+  margin-left: calc(50% - 98px / 2 - 34px);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--second-text-color);
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1:first-child {
+  margin-left: 16px;
+}
+
+.stars--1 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars__reviews {
+  flex-grow: 1;
+  margin-right: 16px;
+  text-align: right;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-color);
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price_word {
+  flex-grow: 1;
+  margin-left: 16px;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--second-text-color);
+}
+
+.card__price_sum {
+  margin-right: 16px;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--text-color);
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  text-align: center;
+  margin: 16px;
+  margin-left: calc(50% - 166px / 2);
```

Comment:
Try to organize your styles 
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)